torch/export/_unlift.py

```diff
@@ -259,16 +259,18 @@ def _create_stateful_graph_module(
     # TODO(suo) this should not be optional, but is since we still ahve
     # capture_pre_autograd_graph grr
     graph_signature: Optional[ExportGraphSignature] = None,
+    _check_input_constraints: bool = True,
```

Comment:
Hmm when do we not want to run this check?

Reply:
I think it's an additional QPS optimization? This is currently something the unflattener also supports, and is turned off in APS [here](https://fburl.com/code/szuarfwu).